packages/synapse-client/out/modelGeneric.mustache

```diff
@@ -0,0 +1,174 @@
+import { mapValues } from '../runtime{{importFileExtension}}';
```

Comment:
Most of this code is copied/generated from the original template source.

Look for `START CUSTOM TEMPLATING FOR synapse-client`